tests/functional/test_project.py

```diff
@@ -96,6 +96,35 @@ def test_get_project_exists(session, a_project):
     )
 
 
+def test_get_project_users(session, a_project):
+    """Test that we can list users associated with a project"""
+
+    # create 2 test users
+    username1 = "test-user-alice"
+    username2 = "test-user-bob"
```

Comment:
I would advise against hard-coding values in tests as that may lead to race conditions or incomplete cleanup that causes issues on the next test run.

You could use the `suffix` fixture and append it to both usernames for some uniqueness.

Reply:
interestingly, that doesn't work

```
  username1 = f"test-user-{suffix}"
  username2 = f"test-user-{suffix}"
```

I think it only makes a single call to `suffix` the fixture and then same `suffix` string is placed in two places here. 

And I learnt that I can't call the fixture directly either :(

Reply:
I created a copy of the fixture `a_user` and called it `b_user` thinking that that it'd deal with setting up and deleting the users for me but that failed too. Both, the original and the copy, fixtures generated the same name.

Reply:
It's okay that the suffix is the same.

If you do `f"test-user-alice-{suffix}"` and `f"test-user-bob-{suffix}"` it will still ensure that both username are unique across test runs.